lib/pinpoint_supported_countries.rb

```diff
@@ -137,7 +137,7 @@ def remap_iso_code(iso_code)
       'BDE' => 'BD',
       'DN' => 'DM',
       'H' => 'HT',
-      'KV' => 'XK',
+      'XV' => 'XK',
```

Comment:
Ugh it's so frustrating the docs went from one wrong code to another wrong code. Thanks for fixing!!